### PR TITLE
Bugfix/fix recovery testsuite

### PR DESCRIPTION
**Pull request by dothebart**

### CODE REVIEW

**Comment by dothebart** [comment]
http://jenkins01.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr/2152/

File: js/client/modules/@arangodb/testsuites/recovery.js [hunk 1]
```diff
@@ -50,75 +50,62 @@ const testPaths = {
...
-    let appDir = fs.join(td, 'app');
+function runArangodRecovery (params) {
+  let argv = [];
+
+  if (params.setup) {
+    params.options.disableMonitor = true;
+    params.td = fs.join(params.tempDir, `${params.count}`);
+    pu.cleanupDBDirectoriesAppend(params.td);
+    let dataDir = fs.join(params.td, 'data');
+    fs.makeDirectoryRecursive(params.td);
```
Comment (jsteemann) [inline js/client/modules/@arangodb/testsuites/recovery.js]: should we create the `dataDir` directory here as well?

File: js/client/modules/@arangodb/testsuites/recovery.js [hunk 2]
```diff
@@ -50,75 +50,62 @@ const testPaths = {
...
-    fs.makeDirectoryRecursive(td);
-    instanceInfo.tmpDataDir = td;
-
-    let appDir = fs.join(td, 'app');
+function runArangodRecovery (params) {
+  let argv = [];
+
+  if (params.setup) {
+    params.options.disableMonitor = true;
+    params.td = fs.join(params.tempDir, `${params.count}`);
```
Comment (jsteemann) [inline js/client/modules/@arangodb/testsuites/recovery.js]: what does `td` stand for?

File: js/client/modules/@arangodb/testsuites/recovery.js [hunk 3]
```diff
@@ -50,75 +50,62 @@ const testPaths = {
...
+    let args = pu.makeArgs.arangod(params.options, appDir, '', tmpDir);
+    // enable development debugging if extremeVerbosity is set
+    if (params.options.extremeVerbosity === true) {
+      args['log.level'] = 'development=info';
     }
+    args = Object.assign(args, params.options.extraArgs);
+    args = Object.assign(args, {
+      'wal.reserve-logfiles': 1,
+      'rocksdb.wal-file-timeout-initial': 10,
+      'database.directory': dataDir + '/db',
```
Comment (jsteemann) [inline js/client/modules/@arangodb/testsuites/recovery.js]: `fs.join()`?

File: js/client/modules/@arangodb/testsuites/recovery.js [hunk 4]
```diff
@@ -133,14 +120,11 @@ function recovery (options) {
     return results;
   }
 
-  let status = true;
```
Comment (jsteemann) [inline js/client/modules/@arangodb/testsuites/recovery.js]: results.status is not initialized anymore now.

**Comment by dothebart** [comment]
http://jenkins01.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr/2160/

**Comment by dothebart** [comment]
http://jenkins01.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr/2165/

**Pull request merged**